index.html

```diff
@@ -438,6 +438,21 @@ <h2 class="photostack-title"><a href="http://nolik2000.github.io." target="_blan
               <h3>Nastya Kostiv</h3>
             </figcaption>
           </figure>
+		<figure>
+         	<img src="https://cloud.githubusercontent.com/assets/9834624/12374282/d01df64a-bc64-11e5-8c5f-df8d9643a333.png" alt="img33" >
```

Comment:
Add image in the `img/gallery` and then access it here. Follow the format @1Niels